platform/solaris/ips/create.sh

```diff
@@ -185,33 +194,68 @@ do
    PKG pkgsend add dir mode=0755 owner=root group=bin path=${dir}
 done
 
-for dir in /var/opengrok /var/opengrok/data /var/opengrok/etc \
-           /var/opengrok/log /var/opengrok/src
+PKG pkgsend add dir mode=0755 owner=webservd group=webservd path=/var/opengrok
```

Comment:
Also, the reason why packages must not deliver under `/var/share` is simple - if a BE A installs a package bits to /var/share and then BE B uninstalls the package and BE A is activated, its world will be corrupted.